youtube_dl/extractor/televizeseznam.py

```diff
@@ -0,0 +1,85 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+
+from .common import InfoExtractor
+from ..compat import compat_urlparse
+
+
+class TelevizeSeznamIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?televizeseznam\.cz/video/.+/(?P<id>.+)'
```

Comment:
`id` is more likely just the numeric part at the end. So more like this:
`r'https?://(?:www\.)?televizeseznam\.cz/(.+\/).+-(?P<id>[0-9]+)'`

Reply:
Well, this is an identifier accepted by televizeseznam.cz API, therefore it make sense to let it be as it is now, at least to me :)

Reply:
It also accepts `/video/bbq-salon/WHATEVER_I_WANT_AND_MAKES_SEO_SENSE-63940306` but that doesn't make it `id`, does it? Also when you examine their graphql query there is a attribute called `dotId` which has exactly that value eg. it is `ID` your `video_id` is in their data on attribute `urlName`. Do as you think, but this will be an issue when merging into master.

Reply:
Ok, you convinced me. I'll change it. Thanks